app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(author: @user, post: @post, text: params[:comment][:text])
+    @comment.save
+    redirect_to user_post_path(@user, @post)
+  end
+end
```

Comment:
- [ ] **[OPTIONAL]** It would be great to use strong parameters for resource creation (Comments, Posts), as it is a good security practice! Here's the [link to the docs](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters). 🤓 🔧